app/spdb/build.sbt

```diff
@@ -207,7 +208,7 @@ def swalker(version: Version) = AppConfig(
     "edu.gemini.auxfile.fits.host"               -> "gsconfig.gemini.edu",
     "edu.gemini.auxfile.other.dest"              -> "/gemsoft/var/data/finder/GSqueue/Finders-Test/@SEMESTER@/@PROG_ID@",
     "edu.gemini.auxfile.root"                    -> "/Users/swalker/.auxfile",
-    "edu.gemini.dataman.gsa.summit.host"         -> "hbffits2.hi.gemini.edu",
+    "edu.gemini.dataman.gsa.summit.host"         -> "cpofits-lv1new.cl.gemini.edu",
```

Comment:
Test server, I think.  I understand that the production server is or will be `fits.cl.gemini.edu`.